### DISCUSSION TITLE
Can i deploy web without using docker?

### DISCUSSION THREAD

**Discussion by tarokeitaro** [jeffvli/feishin#400] [General]
I wanted to deploy the web on my own server, but my mini server is no longer powerful enough to install docker.

**Comment by jeffvli** [comment]
Yes this is possible.

Use the `npm run build:web` command to compile the web app. The static files will be created under `./release/app/dist`.

You can then copy those files over to whatever web server you've installed on your machine.

**Reply by tarokeitaro** [reply]
thanks, I'll try it.

**Reply by tarokeitaro** [reply]
I successfully compiled the web application. Can a website use my Navidrome server by default? sorry, I am new here.

**Comment by tarokeitaro** [comment]
hi, is it possible to set the default Feishin server to my navidrome?
i want Feishin to only ask for a username and password, so that friends don't ask again about the server URL.
![image](https://github.com/jeffvli/feishin/assets/42670754/b018fc69-1230-489d-b053-103cf64486a5)

**Reply by jeffvli** [reply]
There's no way to do this at the moment.

[There's been some preliminary work done for the docker version](https://github.com/jeffvli/feishin/issues/380) but it's not usable yet.

**Reply by kgarner7** [reply]
Without Docker, you would need to also serve a `settings.js` file with the following content:
```JavaScript
"use strict";
window.SERVER_URL="${SERVER_URL}";
window.SERVER_NAME="${SERVER_NAME}";
window.SERVER_TYPE="${SERVER_TYPE}";
window.SERVER_LOCK=${SERVER_LOCK};
```
Replace the values in `${}` with the respective variables
